README.md

```diff
@@ -0,0 +1,29 @@
+# Operator Manifest Verification
+Operator Manifest Verification library (`operator-verify`) verifies static operator files against [Operator-Lifecycle-Manager's](https://github.com/operator-framework/operator-lifecycle-manager) types (`ClusterServiceVersions`, `CustomResourceDefinitions`, and `Package` Manifest yamls).
```

Comment:
- `operator-verify` is the name of the cli tool not the library.
- `provides functions to verify` instead of `verifies`
- only `ClusterServiceVersion` is an OLM type. The rest are part of what constitutes an operator manifest. Please rephrase.

Reply:
For the last point: you said it correctly below https://github.com/dweepgogia/new-manifest-verification/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10